routes/userRoute.js

```diff
@@ -0,0 +1,40 @@
+import express from "express";
+import User from "../models/user.js";
+import wrapAsync from "../utils/wrapAsync.js";
+import passport from "passport";
+
+const userRoute = express.Router();
+
+userRoute.get("/signup", (req, res) => {
+    res.render("users/signup.ejs");
+});
+
+userRoute.post("/signup", wrapAsync(async (req, res) => {
+    try {
+        let { username, email, password } = req.body;
+        const newUser = new User({ email, username });
+        const registeredUser = await User.register(newUser, password);
+        console.log({ registeredUser });
```

Comment:
will this console log the password as well? 
If yes, thn won't this be a security concern

Reply:
changed the consoles